.github/workflows/crates-detector.yaml

```
@@ -0,0 +1,77 @@
+    inputs:
+      base_ref:
+        description: "Base branch to compare against"
+        required: true
+        type: string
+
+jobs:
+  setup-detector:
+    name: Setup Rust & Cache
+    runs-on: ubuntu-latest
+    outputs:
+      cache-hit: ${{ steps.rust-cache.outputs.cache-hit }}
```

actionlint (1.7.12)

[error] property "cache-hit" is not defined in object type {}

(expression)